.github/workflows/ci.yml

```diff
@@ -116,6 +127,35 @@ jobs:
           cd ${GITHUB_WORKSPACE}/build && pwd
           du -hcs _deps/
 
+      - name: Configure CMake for sanitizers build
+        if: matrix.sanitizers == 'Sanitizers'
+        run: |
+          apt -y update
+          apt -y upgrade
+          apt install -y clang
```

Comment:
why do we need this step for sanitizers but not for alpine build that uses clang?

Reply:
lets make this a separate step to just install clang

Reply:
because alpine comes prebundled with clang

> lets make this a separate step to just install clang

IMO this is bad as well. It should be prebundled and should go under `container foundtry` and not as part of this workflow file. See: https://github.com/romange/container-foundry/blob/main/u24.04-dev.Dockerfile

Reply:
https://github.com/romange/container-foundry/pull/14

Reply:
please follow up on this

Reply:
I will, let it be for now